roles/sugarizer/tasks/install.yml

```diff
@@ -153,6 +154,10 @@
     src: sugarizer.service
     dest: /etc/systemd/system
 
+- name: systemctl daemon_reload, so systemd (re)reads sugarizer.service
+  systemd:
+    daemon_reload: yes
+
```

Comment:
Re: apache template stanza in install.yml starts at line lines 161 below.

Reply:
Done.  Thanks.